### PR TITLE
Change notifications for the travis build

### PR DESCRIPTION
**Pull request by miogalang**
Fix the notifications for the travis build

/cc @zendesk/butanding @mmolina 

### References
* JIRA: 

### Risks
* Low. Notification of builds might not be sent out

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -11,3 +11,7 @@ install:
 script:
   - vendor/bin/phpunit --testsuite "Zendesk API Unit Test Suites"
   - vendor/bin/phpcs --extensions=php --standard=PSR2 --report=summary -np src/ tests/
+notifications:
+  email: false
```
Comment (miogalang) [inline .travis.yml]: He will stop emailing me about passed/failed builds

File: .travis.yml [hunk 2]
```diff
@@ -11,3 +11,7 @@ install:
...
+  flowdock:
```
Comment (miogalang) [inline .travis.yml]: It will instead appear in the engineering manila flowdock sidebar

File: .travis.yml [hunk 3]
```diff
@@ -11,3 +11,7 @@ install:
...
+    secure: "iLK+qx1LS206GMddngYGToivo0Tq2rx0WNNFdQNTYSltLm8scEBuAzziqH3paTmfD9f312aV1RbDogLqO/ksDQx4CIfnGE3r/UwGmj+aG2AooqZijLHNITf7mU9dmNH2niNbxM7IwOZpZhd30y1D7U6hZdMCiS7bVxJ5fjdfMSw="
```
Comment (miogalang) [inline .travis.yml]: Encrypted the API key so it is not in the repo.

**Comment by joseconsador** [comment]
:+1:

**Comment by mmolina** [comment]
:+1:

**Pull request merged**